llvm/lib/Target/AArch64/MCTargetDesc/AArch64ELFStreamer.cpp

```diff
@@ -202,6 +202,12 @@ class AArch64ELFStreamer : public MCELFStreamer {
       LastEMS = EMS_None;
 
     MCELFStreamer::changeSection(Section, Subsection);
+
+    // Section alignment of 4 to match GNU Assembler
+    if ((Section->getAlign() < 4) && Section->isText()) {
+      Section->setAlignment(Align(4));
+      emitValueToAlignment(Align(4), 0, 1, 0);
```

Comment:
Can you clarify why you need both the `setAlignment` on the section, and the `emitValueToAlignment` (which will insert bytes until the offset in the section is so aligned)

Am I missing when the latter is needed?

Reply:
This was a leftover from initial testing and I forgot to remove it.  I have now corrected it in the latest patch